src/lib.rs

```diff
@@ -79,14 +98,43 @@ pub fn run(args: Args) {
                 args.filter_arg.clone(),
             );
 
-            SearcherBuilder::new()
-                .multi_line(true)
-                .build()
+            get_searcher(output_mode)
                 .search_path(&matcher, path, printer.sink_with_path(&matcher, path))
                 .unwrap();
         });
 }
 
+fn format_relative_path(path: &Path, output_mode: OutputMode) -> &Path {
+    if output_mode == OutputMode::Vimgrep && path.starts_with("./") {
+        path.strip_prefix("./").unwrap()
```

Comment:
I have no idea if there are higher-level/more canonical ways to "normalize" relative paths (or what that even means)?